tests/Behat/features/element-editor.feature

```diff
@@ -6,45 +6,62 @@ Feature: View types of elements in an area on a page
 
   Background:
     Given I add an extension "DNADesign\Elemental\Extensions\ElementalPageExtension" to the "Page" class
+      And a "group" "AUTHOR group" has permissions "Access to 'Pages' section"
       And a "page" "Blocks Page" with a "Alice's Block" content element with "Some content" content
       And the "page" "Blocks Page" has a "Bob's Block" content element with "Some content II" content
 
-    Given I am logged in with "ADMIN" permissions
-      When I go to "/admin/pages"
+  Scenario Outline: I can see the title and summary of each element
+    Given I am logged in with "<group>" permissions
+    When I go to "/admin/pages"
       And I left click on "Blocks Page" in the tree
-    When I see a list of blocks
-      Then I should see "Alice's Block"
-      And I should see "Bob's Block"
-
-  Scenario: I can see the title and summary of each element
-    Given I see a list of blocks
-    Then I should see "Alice's Block" as the title for block 1
```

Comment:
This looks like a reasonably heavy refactor but I've just moved the login and some preliminary assertions out of the `Background`. It didn't really make sense to be here and trying to relogin as a different user wasn't working for me - I didn't really look much further at that.